src/components/atoms/FormErrors.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { colors } from '../../style/common';
+
+import WobblyText from './WobblyText';
+
+const style = StyleSheet.create({
+  formErrors: {
+    padding: 10,
+  },
+  formErrorText: {
+    color: colors.red3,
+  },
+});
+
+interface FormErrorsProps {
+  errors: Array<string | undefined>;
+}
+const FormErrors: React.SFC<FormErrorsProps> = ({ errors }) => {
```

Comment:
`React.SFC` [is deprecated](https://twitter.com/dan_abramov/status/1057625147216220162), we should use `React.FC` instead!

Reply:
Sounds good.